templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/Tooltip.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    ref="tooltip"
+    class="tooltip-container"
+    :style="{ opacity: positioned ? 1 : 0 }"
+  >
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+// import Helpers from "@/utils/Helpers"
+export default {
+  props: {
+    /**
+     * ID of the DOM element that we want to position this tooltip against.
+     */
+    // refId: {
+    //   type: String,
+    //   required: false,
+    // },
+  },
+  data() {
+    return {
+      positioned: false,
+    }
+  },
+  mounted() {
+    // const container = document.getElementById("cos")
+    // const target = container.querySelector(`#${this.refId}`)
```

Comment:
No commented out code unless it's absolutely necessary in which case it should also explain why it's commented out.